modules/django-social-auth/django_social_auth/setup.py

```diff
@@ -13,6 +13,10 @@ def initialize_options(self):
     name="cb_django_social_auth",
     version="0.1",
     packages=["social_auth"],
-    install_requires=[],
+    install_requires=[
+        'django-allauth==0.51.0',
+        'dj-rest-auth==2.2.5',
+        'djangorestframework==3.14.0'
```

Comment:
djangorestframework==3.14.0
I guess this can break compatibility with other packages in Application. Don't you think it should be like this?
djangorestframework >= 3.12.0

Reply:
Done!